packer/conf/buildkite-agent/scripts/buildkite-agent-lifecycled-handler

```diff
@@ -1,3 +1,21 @@
 #!/bin/bash
 
-service buildkite-agent stop
+set -eu -o pipefail
+eval "$(cat /var/lib/buildkite-agent/cfn-env)"
```

Comment:
The cfn-env file now contains the env var for $BUILDKITE_AGENTS_PER_INSTANCE. 

The changes to this file are so the ASG can hold off spinning down the instance, in the case that any of the agents have a job still running.